qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -37,6 +37,7 @@
 import tritonclient.http as httpclient
 
 EXPECTED_NUM_SPANS = 16
+NO_PARENT_SPAN = "0000000000000000"
```

Comment:
Who sets this? Is it an open telemetry default value?

On quick search I'm guessing otel based on [this](https://opentelemetry.io/docs/instrumentation/cpp/getting-started/), but not finding a spec or mention of the default. Can you link one in a comment above this?

Reply:
It is set here: https://github.com/open-telemetry/opentelemetry-cpp/blob/main/exporters/ostream/src/span_exporter.cc#L78C54-L78C68. I'll add comment

Reply:
Is there a doc that mentions this instead? If not, use permalink to code as lines will change over time on `main`